changes/434.bugfix.md

```diff
@@ -0,0 +1,4 @@
+Remove `optional` keyword from `ProductDef` since it is not a valid keyword.
+
+Also moved the `ModuleDefinitionGrammarTest` and
```

Comment:
Remove this, as this is not relevant for any reader of the change log at all.

Reply:
In fact, please don't do this at all. Please keep the PRs clean.

I've added the change to the develop branch: https://github.com/StevenLooman/magik-tools/commit/9451acbd1b9c2aede98cf90f5640e0b966a7af26